### PR TITLE
Nupkg build fix

### PR DESCRIPTION
**Pull request by vchirikov**
Subj, now `dotnet build -c Release` works as expected (will generate a package with weaver)

### CODE REVIEW

File: Directory.Build.props [hunk 1]
```diff
@@ -29,6 +30,15 @@
     <AllowedOutputExtensionsInPackageBuildOutputFolder>$(AllowedOutputExtensionsInPackageBuildOutputFolder);.pdb</AllowedOutputExtensionsInPackageBuildOutputFolder>
     <NoLogo>true</NoLogo>
     <SuppressNETCoreSdkPreviewMessage>true</SuppressNETCoreSdkPreviewMessage>
+    <ArtifactsDir Condition="'$(ArtifactsDir)'==''">$(SolutionDir)_Build/</ArtifactsDir>
+    <BaseIntermediateOutputPath>$([System.IO.Path]::GetFullPath('$(ArtifactsDir)obj/$(Configuration)/$(MSBuildProjectName)'))/</BaseIntermediateOutputPath>
+    <BaseOutputPath >$([System.IO.Path]::GetFullPath('$(ArtifactsDir)bin/$(Configuration)'))/</BaseOutputPath>
+    <BaseOutputPath Condition="$(MSBuildProjectName.Contains('Tests'))">$([System.IO.Path]::GetFullPath('$(ArtifactsDir)tests'))/</BaseOutputPath>
```
Comment (AlexUstinov) [inline Directory.Build.props]: Can we make all paths consistent?
Here you use '/' (slash) as a separator.

File: Orm/Xtensive.Orm/Xtensive.Orm.csproj [hunk 2]
```diff
@@ -1,8 +1,8 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
+<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <IsPackable>true</IsPackable>
-    <DocumentationFile>..\..\_Build\$(Configuration)\lib\$(TargetFramework)\Xtensive.Orm.xml</DocumentationFile>
-    <OutputPath>..\..\_Build\$(Configuration)\lib\</OutputPath>
+    <DocumentationFile>$(SolutionDir)_Build\$(Configuration)\lib\$(TargetFramework)\Xtensive.Orm.xml</DocumentationFile>
```
Comment (AlexUstinov) [inline Orm/Xtensive.Orm/Xtensive.Orm.csproj]: But here '\' (back slash) is being used

File: Directory.Build.props (hunk 1 above)
Comment (vchirikov) [reply Directory.Build.props]: Yep, bad copy-paste, I will change this.

Comment (vchirikov) [reply Directory.Build.props]: Fixed

**Pull request merged**